cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -1625,12 +1625,201 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             let is_min = op == Opcode::Fmin;
             let output_ty = ty.unwrap();
             ctx.emit(Inst::gen_move(dst, rhs, output_ty));
-            let op_size = match output_ty {
-                types::F32 => OperandSize::Size32,
-                types::F64 => OperandSize::Size64,
-                _ => panic!("unexpected type {:?} for fmin/fmax", output_ty),
-            };
-            ctx.emit(Inst::xmm_min_max_seq(op_size, is_min, lhs, dst));
+            if output_ty.lane_count() <= 1 {
```

Comment:
I've been using the following (not that it makes much of a difference logically but for clarity):
```suggestion
            if !output_ty.is_vector() {
```

Reply:
👍